src/kernel/kmain.c

```diff
@@ -134,6 +139,17 @@ void kmain(uint64_t addr)
     print_ko();
   }
 
+  print_step("loading kernel system font");
+
+  kernel_console_font = psf1_load_font("/fonts/zap-light16.psf");
+  kernel_console_font != NULL ? print_ok() : print_ko();
+
+  if (grub_init_framebuffer(mbi) == false) {
+    PANIC("Failed to initialize framebuffer")
```

Comment:
I guess we cannot load this code earlier but it feels wrong to not display anything on the screen until everything has been set up.

Reply:
Yeah but grub will setup the framebuffer before we even get access to the console...